python/cudf/cudf/io/orc.py

```diff
@@ -395,7 +395,7 @@ def to_orc(
     df,
     fname,
     compression=None,
-    enable_statistics=True,
+    enable_statistics="ROWGROUP",
```

Comment:
To match Parquet API
```suggestion
    statistics="ROWGROUP",
```